bundles/org.openhab.binding.tibber/src/main/java/org/openhab/binding/tibber/internal/config/TibberConfiguration.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.tibber.internal.config;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link TibberConfiguration} class contains fields mapping configuration parameters.
+ *
+ * @author Stian Kjoglum - Initial contribution
+ */
+@NonNullByDefault
+public class TibberConfiguration {
+    private @NonNullByDefault({}) String token;
+    private @NonNullByDefault({}) String homeid;
```

Comment:
I think the new standard for dealing with the null issue is to just give these fields a default value, like an empty string.
```suggestion
    private String token = "";
    private String homeid = "";
```